src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+import { getUserById } from '../../services/user';
+import { completeTodo } from '../../services/todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user = getUserById(todo.userId);
+
+  return (
+    <article
+      className={`TodoInfo ${completeTodo(todo.completed)}`}
```

Comment:
let's use the classnames package instead of this function

Reply:
done